docs/network-iot/iot-denylist.mdx

```diff
@@ -66,13 +66,13 @@ appropriately asserted attributes, further analysis of the Hotspot can take plac
 Community Discord channel,
 [#crowdspot-denylist](https://discord.com/channels/404106811252408320/984427647977283674).
 
-**What if the Denylist doesn't include all of the gaming Hotspots?**
+### What if the Denylist doesn't include all of the gaming Hotspots?
 
 The Denylist is one disincentive of many to keep gaming at bay. Above all, Proof of Coverage and
 more direct detection methods all work in concert to ensure the security of PoC.
 
-**Will the Denylist have the ability to block data transfer or rewards for data transfer?**
+### Will the Denylist have the ability to block data transfer or rewards for data transfer?
 
-Yes, any Hotspot considered ‘denied’ of Proof of Coverage rewards will still be able to act as a
+Yes. Any Hotspot considered ‘denied’ of Proof of Coverage rewards will still be able to act as a
```

Comment:
Might be worth rewriting this section. The answer is a strange pair to the question.
Denylist does not affect data transfer.